### ISSUE TITLE
Directives missing on Arguments

### ISSUE THREAD

**Issue by meierw** [magidoc-org/magidoc#373]
### Describe the bug

I'm not entirely sure if this is a bug or by design, but the wording of the magidoc config documentation makes me think it is indeed a bug. Since the documentation states **"directives will be displayed where they are used"**

```js
      /**
       * The directives that should appear in the documentation.
       * When using SDL introspection, the location and definition of these directives will be displayed where they are used.
       *
       * @default []
       */
      directives: [
```

I have the following schema files:
https://gitlab.com/solveri/signify/-/tree/25-unify-the-validation-messages/_kt/api/src/main/resources/graphql?ref_type=heads

I have the following `magidoc.mjs`
```js
export default {
  introspection: { type: "sdl", paths: ["_kt/api/src/main/resources/graphql/**/*.gqls"] },
  website: {
    template: "carbon-multi-page",
    output: "./dist/apiDocs",
    options: { directives: [{ name: "*", args: ["*"] }] },
  },
}

```

The most noteworthy part is how I have Directives assigned to Arguments, like so:

```graphql
"""
Mutations related to Youtube resources.
"""
type YoutubeMutation {
  """
  Make a claim on a Youtube video, using the `youtubeId` and claim fields.
  """
  makeClaim(
    youtubeVideoId: String! @YoutubeVideoId
    content: String!
      @Pattern(regexp: "^placeholder$", message: "Must be fixed value - 'placeholder'")
    certificate: String!
      @Pattern(regexp: "^placeholder$", message: "Must be fixed value - 'placeholder'")
    signature: String!
      @Pattern(regexp: "^placeholder$", message: "Must be fixed value - 'placeholder'")
  ): String
}
```

But generating the docs, I do not see Directives added to Arguments. I do see them in the left navbar though. And previously I did see them correctly appended to fields where they are used, but not on arguments.

![image](https://github.com/magidoc-org/magidoc/assets/13764651/604f9933-072f-408b-ba38-f4b886cb5369)

### Reproduction

Assuming you have `magidoc` installed globally.

Check out this particular branch in my Open Source repo.

https://gitlab.com/solveri/signify/-/tree/25-unify-the-validation-messages/_kt/api/src/main/resources/graphql?ref_type=heads

Run the following

```
magidoc generate --file=_x/magidoc/magidoc.mjs
magidoc preview --file=_x/magidoc/magidoc.mjs
```

### System Info

```shell
  System:
    OS: Linux 6.8 Pop!_OS 22.04 LTS
    CPU: (16) x64 Intel(R) Core(TM) i7-10875H CPU @ 2.30GHz
    Memory: 5.31 GB / 31.22 GB
    Container: Yes
    Shell: 3.7.1 - /usr/bin/fish
  Binaries:
    Node: 20.12.2 - /nix/store/74a6mmrgxzcg9axl1gmdz3j2y1bjd13f-nodejs-20.12.2/bin/node
    npm: 10.5.0 - /nix/store/74a6mmrgxzcg9axl1gmdz3j2y1bjd13f-nodejs-20.12.2/bin/npm
    pnpm: 9.4.0 - /nix/store/80k8kyb8586wq24x473fpy2l04k9dj0s-corepack-nodejs-20.12.2/bin/pnpm
  Browsers:
    Brave Browser: 126.1.67.116
    Chrome: 126.0.6478.61
```

### Severity

Makes magidoc far less useful to us

**Comment by pelletier197** [comment]
Hello. Thank you for raising this. I will have a look as soon as I get some free time. This may take a couple days. 

I will also correct the wording in the documentation. It is indeed misleading.

**Comment by meierw** [comment]
Thanks!

I hope that this functionality is added. I was super stoked today when I found `magidoc` and it's ability to include Directive information and their usage, which I did not get from other tools. When I saw it on fields I assumed it will work everywhere. So it was quite a downer to see they don't work on arguments.

In our case we want to show all Directives, so that they serve a dual purpose of providing extra functionality, while at the same time also serving as informative bits of documentation. This would allow us to keep our code DRY and not need to repeat stuff, like validation messages in our docs and also in our validation logic.

So far `magidoc` is looking like the best tools for our use-case, but if this Directive logic continues to be missing on arguments then unfortunately it will prevent us from getting the needed value out of it fully.

Looking forward to your findings and updates :slightly_smiling_face:
